### DISCUSSION TITLE
Dumb question: What is the token?

### DISCUSSION THREAD

**Discussion by realskudd** [discord-php/DiscordPHP#1197] [General]
Everything I see in this repo and related docs says to use the token from https://discord.com/developers/applications/. 

The only things I can find are the client ID, client public key, and OAuth2 options. 

Can someone please help me (and others) understand which token all these things are talking about?

Is it the OAuth2 token that's obtained after authenticating?

**Comment by key2peace** [comment]
Your application -> Bot -> the token you got there, only shown once on generation, reset it to get a new one and copy it

![image](https://github.com/discord-php/DiscordPHP/assets/1899472/b243061e-2474-4ba6-ba2a-b4675ef7cecf)

**Reply by realskudd** [reply]
I don't see that button.

![image](https://github.com/discord-php/DiscordPHP/assets/4642150/3168a211-71f2-4303-8141-c2f695c03a5b)

Nowhere in the process was I given the option to copy or generate a token, outside of the OAuth2 section.

**Reply by realskudd** [reply]
🤦 I didn't see the "Bot" section before. I'm an idiot.
